spec/integration/ClientSpec.php

```diff
@@ -29,7 +29,6 @@ function let(){
 
         $this->beConstructedWith([
             'baseUrl'       => getenv('NOTIFY_API_URL'),
-            'serviceId'     => getenv('SERVICE_ID'),
```

Comment:
hey - this is just our integration tests that we run on code change, which use the new format - don't worry, we're not invalidating the old version just yet 🙂